code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -276,6 +276,12 @@
 						and shells to place those soulstones into.</b>"
 	can_repair_constructs = TRUE
 	can_repair_self = TRUE
+	var/health_hud = DATA_HUD_MEDICAL_ADVANCED
```

Comment:
Document this--https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md